src/Symfony/Component/Notifier/Bridge/Discord/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/http-client": "^4.3|^5.0",
-        "symfony/notifier": "~5.3.0"
+        "symfony/notifier": "^5.3"
```

Comment:
Sure, this fixes it, but this defeats the preemptive protection against BC breaks in 5.4...

Reply:
`^5.3,<5.4` ?

Reply:
That doesn't work (same with a conflict rule) or any combination that forbids >=5.4. 
5.x is literally interpreted as 5+ by composer, which may imply 5.4.

Reply:
hmm.. sounds logic.
We can't both accept `5.x` today because it points to `5.3` and refuse `5.x` when it will point to `5.4`.

Is there something to do with branch aliases?